vue-frontend/src/components/home-new/ClientReview.vue

```diff
@@ -0,0 +1,158 @@
+        }"
+        :loop="true"
+        :loopedSlides="50"
+        :spaceBetween="20"
+        :pagination="pagination"
+        class="swiper-container"
+      >
+        <swiper-slide v-for="(client, index) in clients" :key="index">
+          <div class="tw-mx-auto" :class="[client.className]">
+            <div
+              class="tw-font-roboto-medium tw-text-[1.375rem] md:tw-text-[2.25rem] lg:tw-text-[3.125rem] tw-leading-[1.875rem] md:tw-leading-[2.75rem] lg:tw-leading-[3.75rem]"
+              v-html="client.review"
+            ></div>
+            <div
+              class="tw-flex tw-flex-row tw-justify-between tw-mt-11 md:tw-mt-20"
+            >
+              <div class="tw-flex tw-flex-col">
+                <span
+                  class="v2-canopas-gradient-text tw-font-inter-semibold tw-text-[1rem] md:tw-text-[1.625rem]"
+                  >{{ client.name }}</span
+                >
+                <span
+                  class="tw-font-inter-light tw-text-[0.75rem] md:tw-text-[1.25rem] tw-leading-[1.375rem] tw-text-black-core"
+                  >{{ client.designation }}</span
+                >
+              </div>
+              <div class="tw-w-[132px] sm:tw-w-fit">
+                <span
+                  class="sm:tw-justify-between tw-mr-1 md:tw-mr-2 tw-font-inter-light tw-opacity-80 tw-text-[0.5rem] md:tw-text-[0.625rem] tw-leading-[1.375rem]"
+                >
+                  REVIEWED ON
+                </span>
+                <font-awesome-icon
+                  v-for="i in 5"
+                  :key="i"
+                  class="fa-star tw-w-[11px] sm:tw-w-[16px] tw-h-[11px] sm:tw-h-[16px] tw-text-[#FF3D2E]"
+                  icon="star"
+                />
+                <div class="tw-flex tw-flex-row">
+                  <img
+                    :src="reviewImage"
+                    class="tw-mr-1 tw-w-[3.313rem] md:tw-w-[4.25rem] tw-h-[0.938rem] md:tw-h-[1.188rem]"
+                    alt="reviewClutchImage"
+                  />
+                  <span
+                    class="tw-ml-[0.425rem] md:tw-ml-2.5 tw-font-inter-light md:tw-font-inter-regular tw-opacity-80 tw-text-[0.5rem] md:tw-text-[0.625rem] tw-leading-[1.375rem]"
+                  >
+                    5.0 RATINGS
+                  </span>
+                </div>
+              </div>
+            </div>
+          </div>
+        </swiper-slide>
+      </swiper>
+    </div>
+  </div>
+</template>
+
+<script>
+import SwiperCore, { Pagination, Autoplay } from "swiper";
+import { Swiper, SwiperSlide } from "swiper/vue";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import reviewImage from "@/assets/images/clients/v3-client/ReviewImage.svg";
+SwiperCore.use([Pagination, Autoplay]);
+export default {
+  data() {
+    return {
+      reviewImage: reviewImage,
+
+      clients: [
+        {
+          id: 1,
+          className:
+            "md:tw-w-[510px] lg:tw-w-[75%] xl:tw-w-[81%] 2xl:tw-w-[70%]",
```

Comment:
No need to add custom width for all widths. Add some x-padding common to all widths.